src/client/lib/_mapbox/baselayer-control-component.js

```diff
@@ -0,0 +1,7 @@
+import renderVue from '~/lib/render-vue'
```

Comment:
I'm not sure what the purpose of this `baselayer-control-component.js` file is. Would be just as simple to call `renderVue(BaselayerControl, props)` where this file is used, right?

Reply:
I did this to separate the dependency on Vue from the map factory and have it in just one place so that it can be replaced easily